player.class.js

```diff
@@ -0,0 +1,92 @@
+import { c } from "./view.js";
+
+class Player {
+    constructor(x, y){
```

Comment:
tieto sx, sy, sw a sh som nepochopil ani po tom komentari...plus tie hodnoty ako "756.3", ja netusim co to je za hodnota a preco je to prave taka hodnota...je lepsie tie hodnoty zadefinovat ako konstanty s nejakym vystiznym nazvom na zaciatku a potom to priradzovat ako konstantu...ak mas napriklad sw = 756.3 a je to sirka niecoho tak by si spravil konstantu ze nieco_width = 756.3 a potom by bolo this.sw = nieco_width